### ISSUE TITLE
Exception TKCustomMapPins.Image on Android

### ISSUE THREAD

**Issue by po-connor** [TorbenK/TK.CustomMap#12]
Hi Torben,

I got an unhandled exception when I tried to add a pin with a custom image on Android device.
After searching a bit, it seems that a **java.lang.NullPointerException** is thrown in [TKCustomMapRenderer.cs](https://github.com/TorbenK/TK.CustomMap/blob/master/TK.CustomMap/TK.CustomMap.Android/TKCustomMapRenderer.cs) at  line 459 :
`this._markers.Add(pin, this._googleMap.AddMarker(markerWithIcon));`

Here's the exception :

```
01-22 15:13:10.818 I/mono-stdout( 6446): Exception : Java.Lang.NullPointerException: Exception of type 'Java.Lang.NullPointerException' was thrown.
01-22 15:13:10.818 I/mono-stdout( 6446):   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <filename unknown>:0 
01-22 15:13:10.818 I/mono-stdout( 6446):   at Android.Runtime.JNIEnv.CallObjectMethod (IntPtr jobject, IntPtr jmethod, Android.Runtime.JValue* parms) [0x00064] in /Users/builder/data/lanes/2512/d3008455/source/monodroid/src/Mono.Android/src/Runtime/JNIEnv.g.cs:195 
01-22 15:13:10.818 I/mono-stdout( 6446):   at Android.Gms.Maps.GoogleMap.AddMarker (Android.Gms.Maps.Model.MarkerOptions options) [0x0004a] in <filename unknown>:0 
01-22 15:13:10.818 I/mono-stdout( 6446):   at TK.CustomMap.Droid.TKCustomMapRenderer+<AddPin>d__29.MoveNext () [0x0015c] in C:\Users\Patrick\Documents\Projects\TK.CustomMap-master\TK.CustomMap\TK.CustomMap.Android\TKCustomMapRenderer.cs:459 
01-22 15:13:10.818 I/mono-stdout( 6446):   --- End of managed exception stack trace ---
```

This is my code (I used your sample for the test) : 

```
var pin = new TKCustomMapPin
{
    Position = position,
    Title = string.Format("Pin {0}, {1}", position.Latitude, position.Longitude),
    ShowCallout = true,
    IsDraggable = true,
    Image = ImageSource.FromFile("pinImage.png")
};
this._pins.Add(pin);
```

When I try to run tit on iOS simulator, there's no Exception but the pin won't be displayed on the map.

Thanks for your work,
po-connor

**Comment by TorbenK** [comment]
Thanks for reporting.

**Comment by TorbenK** [comment]
Fixed in 1.2.3

**Comment by po-connor** [comment]
It works perfectly. Thanks!
